noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -136,8 +136,8 @@ global INITIAL_L2_BLOCK_NUM: Field = 1;
 global BLOB_SIZE_IN_BYTES: Field = 31 * 4096;
 global ETHEREUM_SLOT_DURATION: u32 = 12;
 // AZTEC_SLOT_DURATION should be a multiple of ETHEREUM_SLOT_DURATION
-global AZTEC_SLOT_DURATION: u32 = ETHEREUM_SLOT_DURATION * 2;
-global AZTEC_EPOCH_DURATION: u32 = 16;
+global AZTEC_SLOT_DURATION: u32 = ETHEREUM_SLOT_DURATION * 3;
+global AZTEC_EPOCH_DURATION: u32 = 32;
```

Comment:
Just pointing out this change :point_up: 
Sooner or later we'll need to build docker images from master

Reply:
This slows down testing, right?

Reply:
per discussion rolled this back - this needs at least a comment about these constants changing for prod